orchagent/portsorch.cpp

```diff
@@ -4631,3 +4651,97 @@ bool PortsOrch::initGearboxPort(Port &port)
     return true;
 }
 
+/*
+ * The following method is called from setVoqInbandIntf() introduced in the following PR:
+ * https://github.com/Azure/sonic-swss/pull/1431
+ */
+bool PortsOrch::addInbandVlan(const string &alias, const string &type)
```

Comment:
Thanks for the comments. PR 1431 is merged. setVoqInbandIntf() can be modified to call this api if the inband type is "vlan". Please rebase and modify setVoqInbandIntf() to make this complete and the comments can be removed.

Reply:
Thanks. I will rebase and remove the comment.